### ISSUE TITLE
PDF of CLA required

### ISSUE THREAD

**Issue by KelseyBlodgett** [cla-assistant/cla-assistant#995]
Hello, I wasn't sure if I should reach out to you or the project, but I am trying to have a CLA executed on behalf of Amazon. If it possible for us to execute a pdf cop[y of the CLA? We cannot process a webform through our Legal system. 

Thanks 
Here is the CLA in question: https://cla-assistant.io/StarRocks/starrocks?pullRequest=22594

**Comment by KharitonOff** [comment]
Hi, you should approach the project owners and ask them for a PDF version of the CLA. There is no way to get a pdf copy of the document via CLA Assistant.

**Comment by KelseyBlodgett** [comment]
If I am able to get a PDF, is there any way to upload it into the CLA Assistant system so that our VP doesn't have to go through the webform?

**Comment by KharitonOff** [comment]
@KelseyBlodgett No, there is no such possibility. You will need to sign the form as well. Or you can ask the owner of the repository to put your organization on the "allowed list". This will skip CLA checks for those PRs, that are coming from your organization (the HEAD of the PR should be in the org, listed on the allowed list)
